internal/providers/pke/pkeworkflow/wait_for_cf_completion_activity.go

```diff
@@ -50,7 +51,7 @@ func (a *WaitCFCompletionActivity) Execute(ctx context.Context, input WaitCFComp
 	describeStacksInput := &cloudformation.DescribeStacksInput{StackName: aws.String(input.StackID)}
 
 	err = cfClient.WaitUntilStackCreateCompleteWithContext(ctx, describeStacksInput)
-	if err != nil {
+	if err = amazon.NewAwsStackFailure(err, input.StackID, cfClient); err != nil {
```

Comment:
do we want to collect errors even if the creation was complete? (the err value from line 54 is not checked)

Reply:
The `err` is checked in the if statement. See the `; err != nil`. If the original `err` is `nil` then `NewAwsStackFailure` returns `nil`